Fix: update aria-label from “Mainbar” to “Main navigation bar” for clarity

ZallaxDev opened the pull request:
Replaced the aria-label `Mainbar` used for the main navigation with `Main navigation bar` to improve clarity and accessibility for
screen reader users, as requested in ticket [#43945](https://mantis.ilias.de/view.php?id=43945).

Additionally, we have included the modification in all languages and it has been specifically translated for:
- German
- Spanish
- French
- Hungarian
- Japanese
- Slovenian

ZallaxDev committed:
Fix: update aria-label from “Mainbar” to “Main navigation bar” for clarity

Replaced the aria-label `Mainbar` used for the main navigation with
`Main navigation bar` to improve clarity and accessibility for
screen reader users, as requested in ticket #43945.

ZallaxDev committed:
update: language variables modified in all languages

oliversamoila commented:
Hello @ZallaxDev  
Thank you for the PR. Would you please update this PR based on Matthias' feedback in https://github.com/ILIAS-eLearning/ILIAS/pull/10497#issuecomment-3612846204 ? 
There are also some conflicts that need to be resolved.
Many thanks to you.

oliversamoila commented:
Hello @ZallaxDev .
In addition to the previous feedback, I would like to ask you to also provide PRs for releases 9, 11 and the trunk.
Thank you very much for your support.

Best regards,
@oliversamoila

ZallaxDev commented:
Hello @oliversamoila,

Thank you very much for your feedback!

We will make the changes you suggested, including those from Matthias, and resolve the conflicts so that we can then apply the changes to the other versions of ILIAS.

We have a question regarding the procedure for integrating the changes into the other versions of ILIAS:
Would it be necessary to perform the PRs corresponding to each version in order to merge all versions at the same time, or can they be merged separately as the changes are ready?

Once again, thank you very much for your comments and your time.

Best regards,
Daniel

Annett7811 reviewed (approved):
Hello!

From an accessibility perspective, this looks good. Thank you very much for your work.

Best regards,
Annett

oliversamoila commented:
Hello @ZallaxDev .
thanks for your question. 
After a little time to settle in, I can say that we will always cherry-pick ourselves. So one PR (here for Release 10) is enough. 

Best regards,
@oliversamoila

juanma1331 committed:
Update mainbar_aria_label timestamps to 2026-02-01
- Resolved conflict in ilias_it.lang: kept Italian translation 'Barra principale'

oliversamoila reviewed (approved):
Hello @matthiaskunkel and @Chris-Squirrel, 
This PR should be accepted and integrated by you. I am therefore assigning it to you. Everything is fine with me here.

@ZallaxDev, can you do another rebase or whatever else is necessary to resolve the conflicts?

Best regards,
Oliver

matthiaskunkel commented:
Hi @Chris-Squirrel,
is it ‘Main navigation bar’ or better ‘Main Navigation Bar’?

Chris-Squirrel commented:
> Hi @Chris-Squirrel, is it ‘Main navigation bar’ or better ‘Main Navigation Bar’?

I would prefer Main Navigation Bar - but isn't this for screen readers? Do they differentiate between sentence and title case?

Annett7811 commented:
Hello @Chris-Squirrel!

For screenreaders, it is irrelevant whether “Main navigation bar” is written in sentence case or “Main Navigation Bar” in title case. Screen readers do not interpret capital letters as emphasis or structural features, but read the text equally.

Best regards
Annett

matthiaskunkel commented:
As long as this lang var is only used for screenreaders, I am fine with it.

abrahammordev committed:
fix: Resolve navigation bar aria name at Italian

abrahammordev commented:
Hi @Chris-Squirrel,

I have successfully resolved the conflicts that arose earlier, so as far as I’m concerned, this is ready to be merged. 👌
Thank you very much!!

Best regards,
Abraham

matthiaskunkel commented:
Branch cannot be rebased. Please resolve confilcts. THen we can merge it.

abrahammordev commented:
Hi! @matthiaskunkel ,

Could you check if the last fixes had indeed resolved all the errors in order to do the rebase. Thanks!

Best regards, Abraham.

matthiaskunkel commented:
LGTM, thanks to @abrahammordev for fixing this issue.
